src/screens/diagramTool/DiagramTool.js

```diff
@@ -1,155 +1,190 @@
-import React, { useRef, useState } from 'react';
-import {ScrollView, Dimensions, View, Animated } from 'react-native'
-import { SafeAreaView } from 'react-navigation';
-import TopNavigation from '../../components/TopNavigation';
-import { Button, Text, AlertDialog } from 'native-base';
+/*
+    The DiagramTool component is used to draw a diagram of a crash. The drawing
+    generation is controlled by the @benjeau/react-native-draw library.
+    Reference the library documentation for detailed explanations of component
+    props.
```

Comment:
Can you describe more of what the diagram tool looks like? I'd note that it brings up a canvas with options for a brush, eraser, different colors, etc. I'd also definitely point out that right now, the tool is setup so that it warns the user before they delete the whole canvas. Also point out the tool requires users to press the "save diagram" button.